scapy/fields.py

```diff
@@ -124,11 +124,22 @@ def i2repr(self, pkt, x):
         return repr(self.i2h(pkt, x))
 
     def addfield(self, pkt, s, val):
-        """Add an internal value  to a string"""
+        """Add an internal value to a string
+
+Copy the network representation of field `val` (belonging to layer `pkt`) to
```

Comment:
Could you keep the indent ?

Reply:
Sure! I was just following convention from `MultipleTypeField` a few lines down